### DISCUSSION TITLE
知识库上传文档很慢是正常的吗？

### DISCUSSION THREAD

**Discussion by qux-bbb** [open-webui/open-webui#6850] [Q&A]
如题，在知识库上传文件很慢，想问下速度慢是不是正常的

**Comment by aomiks** [comment]
上传大于2K的文档就报错，把文档删除一些文字后小于2K就可以上传成功。啥情况呢？管理员设置文档上传大小留空无限制

**Comment by captainIT** [comment]
哪上传的知识库？

**Reply by qux-bbb** [reply]
http://127.0.0.1:8080/workspace/knowledge
